SConstruct

```diff
@@ -1108,7 +1109,7 @@ if env['system_sundials'] == 'y':
 
     # Ignore the minor version, e.g. 2.4.x -> 2.4
     env['sundials_version'] = '.'.join(sundials_version.split('.')[:2])
-    if env['sundials_version'] not in ('2.4','2.5','2.6','2.7','3.0','3.1','3.2','4.0','4.1','5.0'):
+    if LooseVersion(env['sundials_version']) < LooseVersion('2.4') or LooseVersion(env['sundials_version']) >= LooseVersion('6.0'):
```

Comment:
I'd suggest creating a variable for `LooseVersion(env['sundials_version'])` here to shorten this line... the whole point was to make it shorter :joy: You could then reuse that variable on line 1118 in that other `if` statement comparing to `2.4`.

Reply:
I think I then also could safely reuse this shorter variable at line 1610 because it's within  another one `if env['system_sundials'] == 'y'` condition check.

Reply:
The last commit is updated.